CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Alloy Release Notes
 
+### Release 1.8.0
```

Comment:
That's fine. I still update the changelog in all my other projects that have a changelog.